config/urls.py

```diff
@@ -25,6 +31,9 @@
     path("api/", include("config.api_router")),
     # DRF auth token
     path("auth-token/", obtain_auth_token),
+    # JWT auth token
+    path('api/token/', TokenObtainPairView.as_view(), name='token_obtain_pair'),
+    path('api/token/refresh/', TokenRefreshView.as_view(), name='token_refresh'),
```

Comment:
I think we can place these endpoints under `config.api_router` and merge them there

that way we will just define them as `token/` and `token/refresh`

Reply:
Noted